tests/python/examples/simulation/test_mpm99.py

```diff
@@ -0,0 +1,41 @@
+import sys
+
+FRAMES = 100
+
+
+def test_mpm99():
+    from taichi.examples.simulation.mpm99 import dt, initialize, substep
+
+    initialize()
+    for i in range(FRAMES):
+        for s in range(int(2e-3 // dt)):
+            substep()
+
+
+def video_mpm99(result_dir):
+    from taichi.examples.simulation.mpm99 import (dt, initialize, material,
```

Comment:
importing ti from mpm99 seems a bit suspicious :P Is it the `ti` after initialization? 
Does it work if we move imports to the top level?

Reply:
Do you mean moving `import taichi as ti` to the top?

Reply:
yup, I wonder it local import here is required. (If it is we need to explain it in the instruction

Reply:
Only moving `import taichi as ti` to the top is OK. For others we have to locally import them because some actual logic of that example is running during the import.